src/packages/fuelrats_api/v3/mockup.py

```diff
@@ -0,0 +1,101 @@
+import typing
+from json import dumps
+from uuid import UUID
+
+import aiohttp
+import attr
+from loguru import logger
+
+from src.packages.rat import Rat
+from src.packages.rescue import Rescue
+from ._converters import RatConverter, RescueConverter
+from .._base import FuelratsApiABC
+
+
+class ApiError(RuntimeError):
+    ...
+
+
+@attr.s
+class MockupAPI(FuelratsApiABC):
+    # name overrides
+    rat_converter = RatConverter
+    rescue_converter = RescueConverter
+
+    # post init behaviors
+    def __attrs_post_init__(self):
+        self.RAT_ENDPOINT = f"{self.url}/rats"
+        self.RESCUE_ENDPOINT = f"{self.url}/rescues"
+
+    # internals
+    @staticmethod
+    async def _query(method: str, query: str, **kwargs) -> typing.Dict:
+        """
+        Invoke API request
+
+        Args:
+            method: HTTP method
+            query:  HTTP endpoint + parameters
+            **kwargs:  kwargs passed to underlying Aiohttp query
+
+        Returns:
+            API response
+
+        Raises:
+            ApiError: non-2xx response code received
+        """
+        logger.debug("[{method}] {query}", method=method, query=query)
+        async with aiohttp.ClientSession() as session:
```

Comment:
[Suggestion] You'll probably want to use the `raise_for_status` and `timeout` parameters here as well. It will save you from having to check the codes yourself.
By default, I believe `aiohttp` will simply return an empty body for a failed request, rather than raise an exception.
Default timeout is 5 minutes, which is probably far too long for our uses.